cm/adapters/outbound/akafka.py

```diff
@@ -47,14 +48,13 @@ def __init__(
         self._provider = provider
 
     async def publish_sample_updated(
-        self, *, submitter_email: str, sample_id: str
+        self, *, sample_no_auth: models.SampleNoAuth
```

Comment:
I agree that the auth token hash should not be something that should be leaked to the outside. However, that you cannot jsonify it is still a problem. The issue is that you are storing it as bytes but is should rather be a string.